docs/userguides/installing_plugins.md

```diff
@@ -49,6 +49,25 @@ To install plugins individually, run the following command:
 ape plugins install vyper solidity
 ```
 
+You can also install plugins from [requirements files](https://pip.pypa.io/en/latest/reference/requirements-file-format/).
+To do this, set-up your requirements file to include ape plugin package names (including `ape-` prefix):
```

Comment:
Not sure I'm fully on board with this change, basically it'll make two different ways of installing plugins, which both use different naming conventions for the plugins themselves

Perhaps the current way of installing plugins is deficient or overly restrictive.

Also gets me thinking if we'd want to switch over somehow to using `pyproject.toml` for ape config files... but that's a way bigger question

Reply:
There are many ways to install plugins already. You can use requirements files today (I do) and use `pip` to install them. This allows you to use the `ape plugins install` command instead of `pip`

Reply:
Perhaps we can switch the dockerfile to use `pip install` instead of `ape plugins install`

Reply:
Yeah, I like the distinction of `pip install -r requirements.txt` vs. `ape plugins install .` because the later only works with plugins so it's a bit safer, and also has the handling for when plugins are "trusted" vs. "community"